### PR TITLE
[master] (fix) ZIL-5407 Specifying --isolated-server-accounts in localdev shou…

### PR DESCRIPTION
**Pull request by rrw-zilliqa**
…ld not stop it including isolated server accounts.

## Description

Don't have counterintuitive behaviour in localdev (also, this makes z blockchain fail to include these accounts in its builds properly)

## Backward Compatibility

- [X ] This is not a breaking change
- [ ] This is a breaking change

## Review Suggestion

Read code and test.

### Implementation

- [ ] **ready for review**

### Integration Test (Core Team)

- [ ] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

File: scripts/localdev.py.orig
Comment (JamesHinshelwood) [inline scripts/localdev.py.orig:1]: Why does this exist?

File: scripts/localdev.py [hunk 1]
```diff
@@ -502,8 +502,9 @@ def tempo_down(config):
               default='localdev',
               show_default=True,
               help="The test network's name")
-@click.option("--isolated-server-accounts",
-              is_flag=True,
+# Need to specify like this, or passing --isolated-server-accounts will cause isolated server accounts to be disabled
+# - rrw 2023-10-10
+@click.option("--isolated-server-accounts/--no-isolated-server-accounts",
```
Comment (yaron-zilliqa) [inline scripts/localdev.py:507]: Not sure I quite understand this change.

File: scripts/localdev.py.orig
Comment (rrw-zilliqa) [reply scripts/localdev.py.orig:1]: It shouldn't - sorry! Will zap.

File: scripts/localdev.py (hunk 1 above)
Comment (rrw-zilliqa) [reply scripts/localdev.py:507]: If you set default=True and then pass '--isolated-server-accounts', the effect is to turn isolated server accounts off. If you make the option explicit like this , that behaviour doesn't occur. I'll update the comment ..

**Review by JamesHinshelwood — DISMISSED** [review]

**Pull request merged**